transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py

```diff
@@ -23,9 +23,15 @@
 
 def geocode_address(address):
     """Geocode street address into lat/long."""
-    response = geocoder.forward(address)
-    coords = response.json()["features"][0]["center"]
-    return dict(longitude=coords[0], latitude=coords[1])
+    # Add this change to return the correct lat/long for the country Czech Republic (returned by SDMX as Czechia)
+    if address == "Czechia":
+        address = "Czech Republic"
+    # Set the type of address to country in order to return the lat/long of the country Georgia and not the US State!
+    response = geocoder.forward(address, types=["country"])
```

Comment:
why not geo code on ISO 3 as we have dicussed? this is abit of a nasty hack now

Reply:
Yeah I know that, I will work on using ISO3 but it will require a little more work as the REF_AREA is not included in the df and I need to keep both country name for display in the charts and the ISO3 for map... I will do that now.

That was a quick fix for now as Georgia was in the US! :)